erpnext/patches/v13_0/update_old_loans.py

```diff
@@ -24,6 +24,15 @@ def execute():
 	frappe.reload_doc('loan_management', 'doctype', 'loan_repayment')
 	frappe.reload_doc('loan_management', 'doctype', 'loan_repayment_detail')
 	frappe.reload_doc('loan_management', 'doctype', 'loan_interest_accrual')
+
+	#handle type casting for is_cancelled field
+	frappe.db.sql('''UPDATE `tabGL Entry` SET is_cancelled = 
```

Comment:
this is a change in GL entry doctype so doesn't belong in `update_old_loans.py`; either separate patch or reordering of patches is required.